usaspending_api/etl/submission_loader_helpers/program_activities.py

```diff
@@ -0,0 +1,115 @@
+from django.conf import settings
+from django.db import connection
+from usaspending_api.references.models import RefProgramActivity
+
+
+PROGRAM_ACTIVITIES = {}
```

Comment:
Might be too late to change. Something like this might benefit by being a Dataclass instead of a dict